README.md

```diff
@@ -103,7 +103,17 @@ module "kms" {
   key_asymmetric_sign_verify_users       = ["arn:aws:iam::012345678901:role/sign-verify-user"]
 
   # Aliases
-  aliases                 = ["one", "foo/bar"]
+  aliases = ["one", "foo/bar"]
```

Comment:
Maybe change `aliases` to `map(string)` or `any` to satisfy this feature request?